Data/models/History.swift

```diff
@@ -134,6 +110,13 @@ public final class History: NSManagedObject, WebsitePresentable, CRUD {
         
         return all(where: predicate, fetchLimit: 100, context: DataController.viewContext) ?? []
     }
+    
+    public class func fetchAllHistory(_ context: NSManagedObjectContext? = nil, visitedAscending: Bool = false) -> [History] {
+        let predicate = NSPredicate(format: "visitedOn >= %@", History.thisMonth as CVarArg)
```

Comment:
does it fetch all history or last months history?

Reply:
This fetches last months history from old History data and being used while migrating existing history to core history. This is the portion we were showing to our users so it makes sense to only migrate this part.

Reply:
sounds fine to me, although i would update method name or add a documentation comment that it fetches last month only

Reply:
Method name and updated documentation comments
Please check at https://github.com/brave/brave-ios/pull/3703/commits/d643b01dc714abb7710d5d699b766a9c7232ac1a